transmart-schemas/src/main/resources/db/changelog/changes/20190617133319_i2b2demodata_change_visit_pk.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.1" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019  The Hyve B.V.
+  ~  This file is distributed under the GNU General Public License
+  ~  (see accompanying file LICENSE).
+  -->
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="gijs@thehyve.nl" id="20190617133319-0">
+        <comment>
+            Change the primary key of the visit_dimension table.
+        </comment>
+        <dropPrimaryKey schemaName="i2b2demodata" tableName="visit_dimension"/>
+        <modifyDataType schemaName="i2b2demodata" tableName="visit_dimension" columnName="encounter_num" newDataType="serial"/>
```

Comment:
when running the liquibase migration script, I get: 

> liquibase.exception.MigrationFailedException: Migration failed for change set db/changelog/changes/20190617133319_i2b2demodata_change_visit_pk.xml::20190617133319-0::gijs@thehyve.nl:
     Reason: liquibase.exception.DatabaseException: ERROR: type "serial" does not exist
  Position: 103 [Failed SQL: ALTER TABLE i2b2demodata.visit_dimension ALTER COLUMN encounter_num TYPE SERIAL USING (encounter_num::SERIAL)]

Reply:
Hm, apparently that is not possible indeed: https://stackoverflow.com/questions/27307835/how-to-convert-primary-key-from-integer-to-serial.
Maybe for this time change the 00000000000000_i2b2demodata.xml file instead, because hardly anyone uses this yet?

Reply:
but what with the sql update script for postgres in transmart-data? I got a similar issue for it: "ERROR:  type "serial" does not exist".